packages/components/pie-card/test/component/pie-card.spec.ts

```diff
@@ -304,4 +284,80 @@ test.describe('PieCard - Component tests', () => {
             });
         });
     });
+
+    test.describe('Prop: `disabled" and tag="button"`', () => {
+        test('should have the correct attributes', async ({ mount, page }) => {
```

Comment:
Minor: Can you add the attribute the test is expecting to have in the description? e.g `should have the correct attributes: aria-disabled & tabindex`

Reply:
considering we check for more than aria-disabled & tabindex and potentially more attributes in the future, i think it's  if we keep it genric and more conscious 👀  wdyt?

Reply:
It's more for visual tests when they fail the description is useful. Currently the description does not tell you what has failed unless you look at the test itself to then see which attributes are the cause. Also not asking for change for this but usually one test suite should include a single expect (which makes the descriptions easier to handle)

Reply:
makes sense, updated 💯